src/bijectors/simplex.jl

```diff
@@ -315,12 +294,10 @@ function add_simplex_link_adjoint!(
 end
 =#
 
-function simplex_invlink_jacobian(
-    y::AbstractVector{T}, ::Val{proj}=Val(true)
-) where {T<:Real,proj}
-    K = length(y)
+function simplex_invlink_jacobian(y::AbstractVector{T}) where {T<:Real}
+    K = length(y) + 1
     @assert K > 1 "x needs to be of length greater than 1"
-    dxdy = similar(y, length(y), length(y))
+    dxdy = similar(y, K, K - 1)
     @inbounds dxdy .= 0
```

Comment:
Same here, maybe use
```suggestion
    dxdy = fill!(similar(y, K, K - 1), 0)
```
?